influxdb_iox/src/commands/debug/mod.rs

```diff
@@ -36,6 +41,9 @@ enum Command {
 
     /// Interrogate the schema of a namespace
     Schema(schema::Config),
+
+    /// Convert IOx parquet files back into line protocol format
```

Comment:
I *think* the official documentation usually capitalizes Parquet in this sort of context? You're on the team though, so you probably have a better idea of how important this is to the project :)

```suggestion
    /// Convert IOx Parquet files back into line protocol format
```

Reply:
Capitalization is good! I don't know of any particularly stringent style guides, but capitalizing proper names sounds like a good plan to me. Thank you